src/main/resources/assets/quark/lang/pl_pl.json

```
@@ -0,0 +1,944 @@
+ 	"item.quark.magenta_rune": "Karmazynowa runa",
+ 	"item.quark.orange_rune": "Pomarańczowa runa",
+ 	"item.quark.pink_rune": "Różowa runa",
+ 	"item.quark.purple_rune": "Fioletowa runa",
+ 	"item.quark.red_rune": "Czerwona runa",
+ 	"item.quark.white_rune": "Biała runa",
+ 	"item.quark.yellow_rune": "Żółta runa",
+ 	"item.quark.rainbow_rune": "Tęczowa runa",
+	"item.quark.pickarang": "Graborang",
+	"item.quark.diamond_heart": "Diamentowe serce",
+	"item.quark.ancient_tome": "Starożytna księga",
+	"item.quark.frog_leg": "Żabia noga",
+	"item.quark.cooked_frog_leg": "Ugotowana żabia noga",
+	"item.quark.golden_frog_leg": "Złota Żabia Noga",
+	"item.quark.crab_leg": "Krabia noga",
+	"item.quark.cooked_crab_leg": "Ugotowana krabia noga",
+	"item.quark.crab_shell": "Skorupa kraba",
+	"item.quark.root_item": "Jaskiniowy korzeń",
+	"item.quark.dragon_scale": "Smocza łuska",
+	"item.quark.cactus_paste": "Kaktusowa pasta",
+	"item.quark.backpack": "Plecak",
+	"item.quark.ravager_hide": "Skóra dewastatora",
+	"item.quark.soul_compass": "Kompas dusz",
+	"item.quark.bottled_cloud": "Chmura w butelce",
+
+	"item.minecraft.potion.effect.quark.resilience": "Mikstura stabilności",
+	"item.minecraft.splash_potion.effect.quark.resilience": "Miotana mikstura stabilności",
+	"item.minecraft.lingering_potion.effect.quark.resilience": "Trwała mikstura stabilności",
+	"item.minecraft.tipped_arrow.effect.quark.resilience": "Strzała stabilności",
+	"item.minecraft.potion.effect.quark.danger_sight": "Mikstura przewidywania",
+	"item.minecraft.splash_potion.effect.quark.danger_sight": "Miotana mikstura przewidywania",
+	"item.minecraft.lingering_potion.effect.quark.danger_sight": "Trwała mikstura przewidywania",
+	"item.minecraft.tipped_arrow.effect.quark.danger_sight": "Strzała przewidywania",
+	"item.minecraft.potion.effect.quark.resistance": "Mikstura hartu",
+	"item.minecraft.splash_potion.effect.quark.resistance": "Miotana mikstura hartu",
+	"item.minecraft.lingering_potion.effect.quark.resistance": "Trwała mikstura hartu",
+	"item.minecraft.tipped_arrow.effect.quark.resistance": "Strzała hartu",
+	
+	"item.quark.biome_map.snowy_tundra": "Mapa badawcza śnieżnej tundry",
+	"item.quark.biome_map.mountains": "Mapa badawcza gór",
+	"item.quark.biome_map.dark_forest": "Mapa badawcza puszczy",
+	"item.quark.biome_map.desert": "Mapa badawcza pustyni",
+	"item.quark.biome_map.savanna": "Mapa badawcza sawanny",
+	"item.quark.biome_map.swamp": "Mapa badawcza bagna",
+	"item.quark.biome_map.giant_tree_taiga": "Mapa badawcza mega tajgi",
+	"item.quark.biome_map.flower_forest": "Mapa badawcza kwiecistego lasu",
+	"item.quark.biome_map.jungle": "Mapa badawcza dżungli",
+	"item.quark.biome_map.bamboo_jungle": "Mapa badawcza bambusowego lasu",
+	"item.quark.biome_map.badlands": "Mapa badawcza badland",
+	"item.quark.biome_map.mushroom_fields": "Mapa badawcza grzybowej wysp",
+	"item.quark.biome_map.ice_spikes": "Mapa badawcza zamarznięgo oceanu",

```

Oficjalne nazwy biomów, bezpośrednio z `pl_pl.json` gry + poprawki pisowni + bazowa gra ma "mapy badacza" nie badawcze
```suggestion
	"item.quark.biome_map.snowy_tundra": "Mapa badacza zaśnieżonej tundry",
	"item.quark.biome_map.mountains": "Mapa badacza gór",
	"item.quark.biome_map.dark_forest": "Mapa badacza puszczy",
	"item.quark.biome_map.desert": "Mapa badacza pustyni",
	"item.quark.biome_map.savanna": "Mapa badacza sawanny",
	"item.quark.biome_map.swamp": "Mapa badacza bagien",
	"item.quark.biome_map.giant_tree_taiga": "Mapa badacza tajgi wielkich drzew",
	"item.quark.biome_map.flower_forest": "Mapa badacza kwiecistego lasu",
	"item.quark.biome_map.jungle": "Mapa badacza dżungli",
	"item.quark.biome_map.bamboo_jungle": "Mapa badacza bambusowego lasu",
	"item.quark.biome_map.badlands": "Mapa badacza badlandów",
	"item.quark.biome_map.mushroom_fields": "Mapa badacza grzybowych pól",
	"item.quark.biome_map.ice_spikes": "Mapa badacza zamarzniętego oceanu",
```